include/picongpu/plugins/makroParticleCounter/PerSuperCell.hpp

```diff
@@ -106,21 +106,21 @@ namespace picongpu
         typedef MappingDesc::SuperCellSize SuperCellSize;
         typedef GridBuffer<size_t, simDim> GridBufferType;
 
-        MappingDesc* cellDescription;
         std::string notifyPeriod;
         std::string m_filenameExtension = openPMD::getDefaultExtension();
         std::string m_filenameInfix = "_%06T";
 
         std::string pluginName;
         std::string pluginPrefix;
         std::string foldername;
+        MappingDesc* cellDescription;
+
```

Comment:
why did you change the order @psychocoderHPC

Reply:
*offline discussion with @psychocoderHPC:*
The original order caused warnings except for CUDA backends. Since now non-CUDA backends are allowed, changing the order is needed to pass the CI